testing/build.gradle

```diff
@@ -50,7 +50,10 @@ dependencies {
   implementation(
       'androidx.annotation:annotation:1.1.0',
       'androidx.lifecycle:lifecycle-livedata-ktx:2.2.0-alpha03',
+      'androidx.recyclerview:recyclerview:1.0.0',
       'androidx.test.espresso:espresso-core:3.2.0',
+      'androidx.test.espresso:espresso-contrib:3.2.0',
+      'androidx.test.espresso:espresso-intents:3.1.0',
```

Comment:
Why is the intents library needed?

Reply:
intents library is not required in the `testing` module.
Updated the change in new commit.